phpunit.xml

```diff
@@ -22,4 +18,8 @@
         </whitelist>
     </filter>
 
+    <logging>
+        <log type="coverage-clover" target="clover.xml"/>
+        <log type="coverage-html" target="coverage/"/>
+    </logging>
```

Comment:
Guess we could actually remove that. We used to use coveralls for a while. But actually that isn't running anymore for a while already. We can consider adding a coverage service later again...

Reply:
Personally I like using the coverage-html for local development. I'd not need clover, of course.
However, we can of course remove both if you prefer.

Reply:
Locally that's fine. It's simply useless to have travis generate those files at the moment.

Reply:
Of course, fixed in 2cbe2c8fed2d67fe64b588874bcdea7d126a1f7e